File: Baekjoon_Online_Judge/NO_02562/2562.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 01                             | #
# | 최댓값                             | #
# -------------------------------------- #


if __name__ == "__main__":
    result = 0
    result_ind = 0

    for ind in range(1, 10):
        number = int(r_input())

        if number > result:
            result = number
            result_ind = ind

    print(result)
    print(result_ind)

File: Baekjoon_Online_Judge/NO_02562/2562.c
#include <stdio.h>
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 01                             | 
 | 최댓값                             | 
 -------------------------------------- 
*/

int main(){
    int numbers[9], result = 0, result_ind;

    for (int i = 0; i < 9; i ++) scanf("%d", &numbers[i]);

    for(int i = 0; i < 9; i ++){
        if (numbers[i] > result){
            result = numbers[i];
            result_ind = i + 1;
        }
    }

    printf("%d\n", result);
    printf("%d", result_ind);
    return 0;
}